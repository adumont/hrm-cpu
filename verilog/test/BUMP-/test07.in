05 d3
24 5b
50 f2
34 9f
25 da
43 6a
21 f2
23 5b
26 4b
22 74
47 a1
28 93
51 4a
07 0f
40 f4
04 eb
22 51
39 3a
15 58
30 ef
01 36
11 ea
17 ad
51 66
41 04
01 3b
21 f5
17 4a
41 eb
30 11
39 83
49 dc
45 05
22 e4
03 58
50 9e
54 59
58 83
36 5c
02 f5
23 d4
12 f3
48 ee
40 4b
49 0b
05 5a
05 fe
26 38
