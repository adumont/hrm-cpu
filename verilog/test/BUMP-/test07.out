d2
5a
f1
9e
d9
69
f1
5a
4a
73
a0
92
49
0e
f3
ea
50
39
57
ee
35
e9
ac
65
03
3a
f4
49
ea
10
82
db
04
e3
57
9d
58
82
5b
f4
d3
f2
ed
4a
0a
59
fd
37
